pkg/cmd/repo/create/http.go

```diff
@@ -31,11 +31,21 @@ type repoTemplateInput struct {
 }
 
 // repoCreate creates a new GitHub repository
-func repoCreate(client *http.Client, hostname string, input repoCreateInput, templateRepositoryID string) (*api.Repository, error) {
+func repoCreate(
+	client *http.Client, hostname string, input repoCreateInput, templateRepositoryID string,
+) (*api.Repository, error) {
 	apiClient := api.NewClientFromHTTP(client)
 
 	if input.TeamID != "" {
-		orgID, teamID, err := resolveOrganizationTeam(apiClient, hostname, input.OwnerID, input.TeamID)
+		if input.OwnerID == "" {
+			return nil, fmt.Errorf(
+				"specify the fully qualified repository name including the organization name. For example:\n\tgh repo create --team %s org-name/%s",
```

Comment:
I don't think this error message belongs in the API layer. Can you move this check to the `RunE` block of `pkg/cmd/repo/create/create.go`? I think that would be much more appropriate, since we deal with user inputs and flag parsing there.

Reply:
But if you give the name of the repository in the interactive input?
https://github.com/cli/cli/commit/458d5cb42f40385cf44bfadb676901e23962bbb3